packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```
@@ -0,0 +1,195 @@
+        margin={{ top: 1, bottom: 6 }}
+      />
+      <FormFieldsWrapper>
+        <FormField
+          label={
+            <LabelFlexWrapper>
+              <Text as="span" variant="h300" margin={{ left: 2 }}>
+                Amount to withdraw
+              </Text>
+              <Button onClick={handleMaxClick} size="medium" variant="tertiary" _textOnly>
+                Max
+              </Button>
+            </LabelFlexWrapper>
+          }
+          error={errors.amount?.message}
+        >
+          <Input
+            {...register('amount', {
+              valueAsNumber: true,
+              validate: {
+                valid: (value) => {
+                  if (!value) {
+                    return 'Enter amount to transfer.'
+                  }
+                  if (!value || isNaN(value) || value < 0) {
+                    return 'The number of JOY tokens to withdraw has to be an integer and greater than 0 (e.g. 15).'
+                  }
+                  return true
+                },
+                accountBalance: (value) => {
+                  if (value && value > accountBalance) {
+                    return 'Not enough tokens in your account balance.'
+                  }
+                  return true
+                },
+              },
+            })}
+            type="number"
+            nodeStart={<JoyTokenIcon variant="regular" />}
+            nodeEnd={
+              <NumberFormat
+                variant="t300"
+                color="colorTextMuted"
+                format="dollar"
+                value={convertedAmount || 0}
+                as="span"
+              />
+            }
+            placeholder={`${JOY_CURRENCY_TICKER} amount`}
+            error={!!errors.amount}
+          />
+        </FormField>
+        <FormField label="Destination account" error={errors.account?.message}>
+          <Input
+            {...register('account', {
+              validate: {
```

Should we also throw an error if you type the wrong address?